oem/dell/dell.go

```diff
@@ -0,0 +1,52 @@
+package dell
+
+import (
+	"fmt"
+	"github.com/stmcginnis/gofish/redfish"
```

Comment:
Please move this down into its own local import group.

Reply:
done